package.json

```diff
@@ -60,11 +60,11 @@
     "@babel/preset-react": "^7.12.1",
     "@babel/register": "^7.0.0",
     "@storybook/react": "^5.2.6",
-    "autoprefixer": "^7.2.6",
+    "autoprefixer": "^10.0.1",
     "babel-eslint": "^10.1.0",
     "babel-loader": "^8.0.0",
     "chai": "^4.2.0",
-    "css-loader": "^0.28.11",
+    "css-loader": "^5.0.0",
```

Comment:
1. I'm a bit concerned by the major version changes throughout, I don't believe we have anything in place to verify the storybook output and make sure things are still rendering properly.
1. A lot of this is counter to the other PR you've submitted. #60

Reply:
I agree this PR seems conterintuitve to #60. I would much rather remove and upgrade items there as need be.